src/components/definition-list/dt.d.ts

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
-
-export interface DtProps {
+import { SpacingProps } from '../../utils/helpers/options-helper';
+export interface DtProps extends SpacingProps{
```

Comment:
```suggestion
export interface DtProps extends SpacingProps {
```
TSLint detects that as an error